Models/Settings/SettingsObject.cs

```diff
@@ -83,6 +78,21 @@ public string DefaultOutputROMFilename
         /// </summary>
         public bool GeneratePatch { get; set; }
 
+        /// <summary>
+        /// Apply patch file during output generation
+        /// </summary>
+        public bool ApplyPatch { get; set; }
+
+        /// <summary>
+        /// Checks if the InputROM is needed for generation
+        /// </summary>
+        public bool NeedInputROM { get { return ApplyPatch | OutputN64ROM | OutputVC | GeneratePatch; } }
```

Comment:
Can be simplified:

`public bool NeedInputROM => ApplyPatch | OutputN64ROM | OutputVC | GeneratePatch;`